pkg/artifacts/paths.go

```diff
@@ -18,14 +18,14 @@ package artifacts
 
 import (
 	"fmt"
-	"os"
-	"path/filepath"
-
 	"github.com/spf13/pflag"
 	"k8s.io/klog"
+	"os"
+	"path/filepath"
 )
 
 var baseDir string
+var runDir string
```

Comment:
one of these days we should really ditch the globals, but this was already the pattern

Reply:
@BenTheElder Can we leave this as globals for now?